packages/frontend/src/bootstrap.tsx

```diff
@@ -24,6 +24,18 @@ import {
     standaloneProviders,
     standaloneSupportedChains,
 } from "./standalone-setup";
+import { QueryClient } from "@tanstack/react-query";
+
+const queryClient = new QueryClient({
+    defaultOptions: {
+        queries: {
+            cacheTime: 1_000 * 60 * 60 * 24, // 24 hours
+            networkMode: "offlineFirst",
+            refetchOnWindowFocus: false,
+            retry: false,
```

Comment:
In offline first mode it will run the `queryFn` once, but then pause retries. 

why do we need `retry` `false` here?

Reply:
I actually just copied over the config from Wagmi. Thought they would use some sensible configuration themselves, so mostly took it. I'm not sure the retries feature is suuuuper useful to us, but I can enable it if needed.